### DISCUSSION TITLE
Slider upon push

### DISCUSSION THREAD

**Discussion by kristianolofsson** [Blackymas/NSPanel_HA_Blueprint#486] [General]
I now see that in version 3.2 a specfic page can be called. But what if i want the slider to show alligned with the physical button?
I have assisgned the physical button to a light also present in page1. If i push it a see the slider. But how to call it from the button?

**Comment by Blackymas** [comment]
long press at HW Buttons

**Reply by kristianolofsson** [reply]
What about this type of solution?
https://github.com/DeanoXX/esphome-config/blob/main/esp_dim09.yaml

Id like to use longpress to dim my lights. Holding button ->Dim up. Release and hold again -> Dim down.

**Reply by Blackymas** [reply]
is not possible with our configuration or I don't see any reason to change it.

**Comment by deejaybeam** [comment]
You have the possibility to create your own custom actions to the buttons.
This way it may be possible to create this with an repeat action, but it´s on your own.
Otherwise it is also possible to call a script, or automation and do (dimm) until you hold the button.

So you have all possibilities - but it´s on you to create your own script/automation to do what your special need is.

**Reply by kristianolofsson** [reply]
Thanks! I will give it a try.
